nix-test/src/const.c

```diff
@@ -209,6 +209,31 @@ get_int_const(const char* err) {
     GET_CONST(ECANCELED);
 #endif
 
+#if defined(__OpenBSD__)
```

Comment:
As we do conversions of our code to use upstream libc values, we can remove these tests incrementally until they're all gone. But for now, we should add these tests if we have any hard-coded values (like all the `errno` values currently are).